saleor/discount/models.py

```diff
@@ -300,6 +301,12 @@ class Promotion(ModelWithMetadata):
 
     class Meta:
         ordering = ("name", "pk")
+        permissions = (
+            (
+                DiscountPermissions.MANAGE_DISCOUNTS.codename,
+                "Manage sales and vouchers.",
```

Comment:
Just wondering if we should keep `sales` name here instead of `Manage promotions and vouchers.` ?

Reply:
This name is stored in the database. It could be fixed in a separate PR.